sonic_data_client/events_client.go

```diff
@@ -332,7 +332,8 @@ func get_events(evtc *EventClient) {
                                 JsonIetfVal: jv,
                             }}
                         if err := send_event(evtc, evtTv, evt.Publish_epoch_ms); err != nil {
-                            break
+                            check_evtc_stopped(evtc)
```

Comment:
Why not check the result? Can you explain?

Reply:
Because we will return if there is an error regardless whether or not event client has been stopped. In the other call to this function we check if event client has been stopped becuase if it has then we will also break out. With the call to this function, we mainly just want to see if event client has been stopped, deinit subscriber.

Reply:
If event client is not stopped, when will you deinit subscribe?

Reply:
We only deinit subscriber when event client is stopped, as the subscriber will subscribe to keep listening for events for as long as they want to